Jenkinsfile

```diff
@@ -8,36 +8,39 @@
 // that one has to allocate a node first, which would stay busy until the end of
 // the parallel pipeline
 
-void buildOnYoctoNode(String variant, String image, boolean checkSmokeTests = false) {
+void buildOnYoctoNode(String variant, String image, boolean checkSmokeTests = false, boolean checkBitbakeTests=false) {
```

Comment:
I think I had issues before when I tried to use more than one optional. If there is one out of two unset variables in a row it should be impossible for the script to know which one is not set.